Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/Nodes/MaterialTemplates/ExperimentalPBR/MaterialGraphName_ForwardPass.azsl.template

```diff
@@ -83,7 +80,10 @@ VSOutput MaterialGraphName_MainPassVS(VSInput IN)
     #undef O3DE_MC_BITANGENT
     #undef O3DE_MC_WORLD_POSITION
 
-    VertexHelper(IN, OUT, worldPosition + inPositionOffset.xyz);
+    OUT.m_worldPosition = worldPosition;
+    OUT.m_position = mul(ViewSrg::m_viewProjectionMatrix, float4(OUT.m_worldPosition, 1.0));
+    OUT.m_normal = IN.m_normal;
+    OUT.m_tangent = IN.m_tangent;
```

Comment:
Even though it's experimental, this is a minor change in behavior with the removal of inPositionOffset. Can you add it back to world position? Otherwise looks good.